pkg/make_deb.py

```diff
@@ -303,15 +303,30 @@ def CreateChanges(output,
           'Checksums-Sha256',
           '\n' + ' '.join([checksums['sha256'], debsize, deb_basename]))
   ])
-  with open(output, 'w') as changes_fh:
+  with open(output, 'wb') as changes_fh:
     changes_fh.write(changesdata.encode('utf-8'))
 
 
 def GetFlagValue(flagvalue, strip=True):
```

Comment:
Nit: The name's a bit vague. Maybe rename to NormalizeUnicode or NormalizeString or something? The docstring should clarify what specific kind of normalization we're talking about.

Reply:
I want to keep the name of GetFlagValue just to limit the number of changed lines.
But I did expand the docstring and substantially re-wrote it to do the fsencoding stuff here, so that we do not (incorrectly) turn a utf-8 path in @filename args to an improper path.